classes/village/VillageManager.php

```diff
@@ -1682,7 +1712,7 @@ public static function enactProposal(System $system, User $player, int $proposal
             $message .= "\n You have gained {$rep_adjustment} Reputation!";
             $player->updateData();
         } else if ($rep_adjustment < 0) {
-            $player->reputation->subtractRep($rep_adjustment, UserReputation::ACTIVITY_TYPE_UNCAPPED);
+            $player->reputation->subtractRep((-1 * $rep_adjustment), UserReputation::ACTIVITY_TYPE_UNCAPPED);
```

Comment:
If this has the possibility of being anything but a positive integer, I think using `abs()` would be more appropriate. It could have unintended result in the `subtractRep()` method.
![image](https://github.com/elementum-games/shinobi-chronicles/assets/24821372/695e1e77-854c-468f-b874-a4ff1d357676)

Reply:
Makes sense to me!